src/marker-layer.coffee

```diff
@@ -19,13 +19,7 @@ class MarkerLayer
     store
 
   @serializeSnapshot: (snapshot) ->
```

Comment:
This method has now become a no-op, but I believe it's reasonable to keep it for symmetry with `@deserializeSnapshot` and, more importantly, because the way we serialize stuff is an implementation detail.